### PR TITLE
[CST-271] add notes field to admin console

### PR DESCRIPTION
**Pull request by xswoodsx**
## Ticket and context

Ticket: [271](https://dfedigital.atlassian.net/jira/software/projects/CST/boards/119?assignee=5f3543bb3e9e2e004db3189d&selectedIssue=CST-271)

## Tech review
Screenshot 2022-01-19 at 07 43 59
Screenshot 2022-01-19 at 07 44 20
Screenshot 2022-01-19 at 07 44 28

[Prototype PR with screenshots](https://github.com/DFE-Digital/early-career-framework-register-prototype/pull/243)

### Code quality checks
- [ ] All commit messages are meaningful and true
- [ ] Added enough automated tests

### How can someone see it it review app?
1. Click the link to review app (posted by a `github-actions` bot below)
2. Login as `admin@example.com`
3. Go to participants dashboard, either for a school or the whole list
4. Select a ECT or Mentor
5. Add and save a note
6. Edit the note

### CODE REVIEW

File: app/views/admin/notes/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<% content_for :title, "Participant notes" %>
+<% content_for :before_content, govuk_back_link(text: "Back", href: admin_participant_path) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+   <span class="govuk-caption-l"><%= @participant_profile.user.full_name %></span>
+    <%= form_for @participant_profile, url: admin_note_path, method: :put do |f| %>
```
Comment (tonyheadford) [inline app/views/admin/notes/edit.html.erb:7]: I think if you used `form_for @participant_profile, as: :participant_profile` then you could avoid all the `profile_type` stuff in the `note_params` method and just do `params.require(:participant_profile).permit(:notes)`

**Review by tonyheadford — COMMENTED** [review]
LGTM - just one comment regarding the form params

File: app/views/admin/notes/edit.html.erb (hunk 1 above)
Comment (xswoodsx) [reply app/views/admin/notes/edit.html.erb:7]: ahhhh ok, brill, yeah, that profile type is annoying, i'll switch it to your suggestion.

**Review by tonyheadford — APPROVED** [review]
Nice one!

**Pull request merged**